legislation-parameters/README.md

```diff
@@ -1,60 +0,0 @@
-# Legislation parameters
```

Comment:
You need to update the [SUMMARY.md](https://github.com/openfisca/openfisca-doc/blob/master/SUMMARY.md) when you change the file system structure. 

Otherwise it doesn't work :
![image](https://cloud.githubusercontent.com/assets/11834997/25493093/56ea4e0a-2b75-11e7-802f-a4be3e065f5a.png)